sdk/core/core-http/src/policies/throttlingRetryPolicy.ts

```diff
@@ -46,7 +48,10 @@ export class ThrottlingRetryPolicy extends BaseRequestPolicy {
 
   public async sendRequest(httpRequest: WebResourceLike): Promise<HttpOperationResponse> {
     return this._nextPolicy.sendRequest(httpRequest.clone()).then((response) => {
-      if (response.status !== StatusCodes.TooManyRequests) {
+      if (
```

Comment:
Replace `.then()` call back with await?

Reply:
oh please ignore I think Harsha already did that in his PR.

Reply:
Which PR?

Reply:
#15906 but it looks like he only updates the `.then()` call back of `delay()`, not this. so I suggest replacing with
```ts
const response = await this._nextPolicy.sendRequest(httpRequest.clone());
...
```

Reply:
On it!